src/main/java/trackitnus/commons/core/Messages.java

```diff
@@ -19,4 +19,9 @@
     public static final String MESSAGE_MODULE_DOES_NOT_EXIST = "The module specified in the command doesn't exist.";
     public static final String MESSAGE_TASK_DOES_NOT_EXIST = "There is no such task in the app";
 
+    public static final String MESSAGE_DUPLICATE_CONTACT = "This contact already exists in the app";
+    public static final String MESSAGE_DUPLICATE_LESSON = "This lesson already exists";
+    public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
+    public static final String MESSAGE_DUPLICATE_MODULE = "This module already exists";
+    public static final String MESSAGE_DUPLICATE_TASK = "This task already exists in the app";
```

Comment:
I think for the messages about duplicates, all should end with `already exists`, no need `already exists in the app`.

Reply:
Fixed!